docs/designers-developers/developers/data/data-core-block-editor.md

```diff
@@ -775,6 +775,20 @@ via its `onChange` callback, in addition to `onInput`.
 
 Whether the most recent block change was persistent.
 
+### __unstableIsLastBlockChangeIgnored
```

Comment:
hmm... I didn't think __unstable or __experimental was supposed to show up in docs?

Reply:
> hmm... I didn't think __unstable or __experimental was supposed to show up in docs?

It caught me off guard as well. I think they may only be omitted in documentation generated by the [`docgen` tool](https://github.com/WordPress/gutenberg/tree/master/packages/docgen). The data modules documentation is a separate process. It should be improved (I'll create a follow-up task).